remotespark/RemoteSparkMagics.py

```diff
@@ -49,11 +46,6 @@ def __init__(self, shell, data=None, test=False):
         except KeyError:
             self.logger.error("Could not read env vars for serialization.")
 
-        if use_auto_viz:
```

Comment:
use_auto_viz is a configuration that is not needed anymore. Let's remove all traces of it.